WORKSPACE

```diff
@@ -3502,2335 +3510,2407 @@ rules_proto_dependencies()
 )
 
 rpm(
-    name = "xkeyboard-config-0__2.28-1.el8.x86_64",
-    sha256 = "a2aeabb3962859069a78acc288bc3bffb35485428e162caafec8134f5ce6ca67",
-    urls = ["http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/xkeyboard-config-2.28-1.el8.noarch.rpm"],
+    name = "xkeyboard-config-0__2.33-2.el9.x86_64",
+    sha256 = "ca47ef1bfc9cf8b0996ffad8c423270e84f87fb2a32386b03edadc5d38a2fdf5",
+    urls = ["https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/xkeyboard-config-2.33-2.el9.noarch.rpm"],
 )
 
 rpm(
-    name = "xml-common-0__0.6.3-50.el8.x86_64",
-    sha256 = "6d7676847b3c0dbac22983c85c0a419af43029cc3b8ff5dc26c9f85174fc85d8",
-    urls = ["http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/xml-common-0.6.3-50.el8.noarch.rpm"],
+    name = "xz-0__5.2.5-8.el9.x86_64",
+    sha256 = "159f0d11b5a78efa493b478b0c2df7ef42a54a9710b32dba9f94dd73eb333481",
+    urls = ["https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/xz-5.2.5-8.el9.x86_64.rpm"],
 )
 
 rpm(
-    name = "xz-0__5.2.4-4.el8.x86_64",
-    sha256 = "99d7d4bfee1d5b55e08ee27c6869186531939f399d6c3ea33db191cae7e53f70",
-    urls = ["http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/xz-5.2.4-4.el8.x86_64.rpm"],
+    name = "xz-libs-0__5.2.5-8.el9.x86_64",
+    sha256 = "ff3c88297d75c51a5f8e9d2d69f8ad1eaf8347e20920b4335a3e0fc53269ad28",
+    urls = ["https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/xz-libs-5.2.5-8.el9.x86_64.rpm"],
 )
 
 rpm(
-    name = "xz-libs-0__5.2.4-4.el8.x86_64",
-    sha256 = "69d67ea8b4bd532f750ff0592f0098ace60470da0fd0e4056188fda37a268d42",
-    urls = ["http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/xz-libs-5.2.4-4.el8.x86_64.rpm"],
+    name = "yajl-0__2.1.0-21.el9.x86_64",
+    sha256 = "d159334f408022942e77f67322288d13c1d575a3af54512d4310310709b644d9",
+    urls = ["https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/yajl-2.1.0-21.el9.x86_64.rpm"],
 )
 
 rpm(
-    name = "yajl-0__2.1.0-11.el8.x86_64",
-    sha256 = "55a094ffe9f378ef465619bf6f60e9f26b672f67236883565fb893de7675c163",
-    urls = ["http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/yajl-2.1.0-11.el8.x86_64.rpm"],
+    name = "zip-0__3.0-33.el9.x86_64",
+    sha256 = "adcd60a331a0a31ad0d36fcec522203b58151aff41689faede895a42529b3f87",
+    urls = ["https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/zip-3.0-33.el9.x86_64.rpm"],
 )
 
 rpm(
-    name = "zlib-0__1.2.11-19.el8.x86_64",
-    sha256 = "439833454c91b662c1ed99eff65e4726d765e974f65faadaf1c29eb1281f28f9",
-    urls = ["http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/zlib-1.2.11-19.el8.x86_64.rpm"],
+    name = "zlib-0__1.2.11-35.el9.x86_64",
+    sha256 = "80df42ac4be2c057e332647ca98d65b1548d4e4adf52beb75d79e79fc8e48aa7",
+    urls = ["https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/zlib-1.2.11-35.el9.x86_64.rpm"],
```

Comment:
how come that everything is changed from el8 to el9? I'd expect the el9 dependencies to be added in addition to the el8 ones as we keep building virt-v2v on el8, no?

Reply:
i thought you wish to have el9 also for virt-v2v, i'll add back el8.